content/customize-gnome.md

```diff
@@ -18,117 +18,141 @@ section: software
+**NOTE:** There are many extensions to choose from! If one isn't currently working, or breaks with an update, then try a different extension to see if it works better.
 
 ---
 
-## User Themes
+## Included Extensions
 
-[User Themes](https://extensions.gnome.org/extension/19/user-themes/)
+Pop!\_OS includes the GNOME Extensions app by default. Which lists both system-installed, and user-installed extensions.
 
-This extension allows for the desktop to be themed in further ways than default.  Many themes and extensions can be installed from [opendesktop.org](https://www.opendesktop.org/s/Gnome), among others.  This extension is installed by default in Pop!_OS, and can be added to Ubuntu 18.04.  
+![extensions-app-window](/images/customize-gnome/extensions-app-window.png)
 
----
+Each extension has options for customizing settings (if supported in Extension metadata), or viewing the webpage for the extension to learn more information. The Extensions app also allows you to quickly enable and disable installed extensions.
 
-## Dash to Dock
+Pop!\_OS includes the following extensions by default. In most cases, these extensions should be left **enabled**.
 
-[Dash to Dock](https://extensions.gnome.org/extension/307/dash-to-dock/)
+### [Cosmic Dock](https://github.com/pop-os/cosmic-dock)
 
-![dash-to-dock](/images/customize-gnome/dash-to-dock.png)
+A fork of Dash to Dock customized for Pop!\_OS.
 
-This extension brings a launcher similar to the Unity launcher on the left side, or the macOS launcher on the bottom.  It is high customizable and allows for different locations, styles, sizes, and layouts.  This is included by default in Ubuntu 18.04 and can be added to Pop!_OS.
+![cosmic-dock-close](/images/customize-gnome/cosmic-dock-close.png)
 
----
+**NOTE:** It is **not** recommended to have both Cosmic Dock and Dash-t0-Dock installed an enabled at the same time. User Interface (UI) bugs will occur. Choose one or the other.
 
-## Backslide
+### [Cosmic Workspaces](https://github.com/pop-os/cosmic-workspaces)
 
-[Backslide](https://extensions.gnome.org/extension/543/backslide/)
+A fork of this [extension](https://github.com/RensAlthuis/vertical-overview) designed to return the GNOME workspaces to a vertical layout rather than the horizontal view introduced in GNOME 40.
 
-![backslide](/images/customize-gnome/backslide.png)
+Workspace behaviors can be edited in Settings.
 
-This extension is a wallpaper slideshow for your desktop.  After choosing a folder of pictures, they will be rotated automatically at a predefined time period.
+![workspaces-and-settings](/images/customize-gnome/workspaces-and-settings.png)
 
----
+### [Desktop Icons NG (DING)](https://gitlab.com/rastersoft/desktop-icons-ng)
 
-## Window List
+Since Desktop icons have been deprecated in mainline GNOME, Pop!_OS includes a GNOME extension to return this feature.
```

Comment:
Maybe this should be "to bring this feature back"?

Reply:
Fixed.